packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -140,14 +140,24 @@ const CFDsListing = () => {
                         const has_mt5_account_status = existing_account.status
                             ? getMT5AccountAuthStatus(existing_account.status)
                             : null;
+                        const is_demo_account = existing_account.account_type === 'demo';
+                        const is_demo_get_account = is_demo && existing_account.action_type === 'get';
                         return (
                             <TradingAppCard
                                 action_type={existing_account.action_type}
                                 availability={selected_region}
                                 clickable_icon
                                 icon={existing_account.icon}
-                                sub_title={existing_account?.sub_title}
-                                name={!has_mt5_account_status ? existing_account?.name : ''}
+                                sub_title={
+                                    is_demo_account
+                                        ? localize('{{sub_title}} Demo', { sub_title: existing_account?.sub_title })
+                                        : localize('{{sub_title}}', { sub_title: existing_account?.sub_title })
+                                }
+                                name={
+                                    !has_mt5_account_status && is_demo_get_account
```

Comment:
same here. check for is_demo to see if you need to show 'Demo' or not

Reply:
For this we need to check if its demo and it doesnot have a multi-action  action_type

Reply:
Done